src/commands/login.rs

```diff
@@ -256,10 +264,29 @@ impl LoginCommand {
         let login_connection = match self.auth_method() {
             AuthMethod::Github => self.run_interactive_gh_login().await?,
             AuthMethod::UsernameAndPassword => self.run_interactive_basic_login().await?,
+            AuthMethod::Token => self.login_using_token().await?,
```

Comment:
I have probably misread but currently `auth_method()` looks like it only returns `Token` if the user passes `--auth-method token` on the command line.  Do we not infer it from the `--token` option?  (I guess that could cause issues if the user wanted to override an ambient EV with a different login option... but...)  Again, I have probably misunderstood, so please correct me!